### PR TITLE
[10.0][PORT] pos_margin

### PR DESCRIPTION
**Pull request by legalsylvain**
Trivial PR.

Port pos_margin from 8.0 to 10.0. 
Only the last commit is to review.  	6f5e923

Thanks.

CC : @hparfr, @bealdav.

### CODE REVIEW

**Comment by bealdav** [comment]
LGTM 👍

**Comment by hparfr** [comment]
👍

**Comment by rafaelbn** [comment]
Hi, please take a look on this issue: https://github.com/OCA/pos/issues/270 Thanks :smiley:

File: pos_margin/README.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
+This module adds the 'Margin' on sales order.
+=============================================
```
Comment (chienandalu) [inline pos_margin/README.rst:2]: I'd say the issue comes from this displaced title

**Pull request merged**